main.go

```diff
@@ -114,6 +114,8 @@ func main() {
 				return err
 			}
 			logrus.SetOutput(f)
+		} else {
+			logrus.SetOutput(os.Stdout)
```

Comment:
I am thinking we should follow the advice of @mfriedenhagen in [sysbox issue #221](https://github.com/nestybox/sysbox/issues/221)  and log to `os.stderr` instead.  This is a more appropriate channel for logging and allows us to use `os.stdout` for other purposes in the future.

Reply:
Agree, let's go with os.stderr, but let's identify the issue with its sibling sysbox-fs PR before merging this one.

Reply:
Sorry, i forgot about that! I did a git amend to keep a single commit.